tests/unit/s3middleware/objectLegalHold.js

```diff
@@ -1,11 +1,13 @@
 const assert = require('assert');
-const { parseLegalHoldXml } = require('../../../lib/s3middleware/objectLegalHold');
+const { convertToXml, parseLegalHoldXml } =
+    require('../../../lib/s3middleware/objectLegalHold');
 const DummyRequestLogger = require('../helpers').DummyRequestLogger;
 
 const log = new DummyRequestLogger();
 
 function generateXml(status) {
```

Comment:
Since you only use this in one describe block, if you move this into the describe block it will be closer to its use cases.

Reply:
It is used in both, but moved bit closer.